### ISSUE TITLE
Scan panel shows wrong wireframes for system Ooscs Freau MS-U f2-2283

### ISSUE THREAD

**Issue by EoD** [EDDiscovery/EDDiscovery#2193]
While only the main star was scanned, it looked fine. After scanning the first planet, the wireframe "A" and "Belt" appeared.

![image](https://user-images.githubusercontent.com/293499/47556657-36af6000-d90f-11e8-83db-b1662b1e9a96.png)

When I disable "EDSM", then the "Belt" disappears.

https://www.edsm.net/en/system/id/5822670/name/Ooscs+Freau+MS-U+f2-2283

**Comment by klightspeed** [comment]
The red wireframe suggests that somewhere along the line it is being added as a star rather than as a belt.  What information does it present when you click on the "Belt" body?

**Comment by klightspeed** [comment]
Would it please be possible to get a copy of the journal where this occurred so we can diagnose what happened?

**Comment by robbyxp1** [comment]
Close no response

**Comment by EoD** [comment]
Sorry, I missed the reply on Discord when I asked about it. I sent the file to klightspeed.

**Comment by klightspeed** [comment]
```
{ "timestamp":"2018-10-26T09:02:23Z", "event":"SupercruiseExit", "StarSystem":"Ooscs Freau MS-U f2-2283", "SystemAddress":1225806840925, "Body":"Ooscs Freau MS-U f2-2283 A Belt", "BodyID":1, "BodyType":"StellarRing" }
```

As I suspected, you dropped out in the belt, and as `StellarRing` bodies were not filtered out it tried to parse the name -> `Ooscs Freau MS-U f2-2283`/`A`/`Belt`

This should have been fixed by #2194
